### ISSUE TITLE
Pause Stream?

### ISSUE THREAD

**Issue by hiteshsatwani** [yivanscreations/lrc#1]
Is it possible to pause the current lyrics stream?

**Comment by Yivan000** [comment]
Yeah adding the function for pausing the stream needs a full rewrite of the streaming code, which it needs since the current one is rather cobbled together at the last minute. I will get to it at some point.
